inspirehep/utils/tickets.py

```diff
@@ -24,23 +24,36 @@
 
 from __future__ import absolute_import, division, print_function
 
+from functools import wraps
 from urlparse import urljoin
 
 from flask import current_app, render_template
 
-from rt import ALL_QUEUES, ConnectionError
+from rt import ALL_QUEUES, ConnectionError, AuthorizationError
 from .proxies import rt_instance
 
 
 class EditTicketException(Exception):
     pass
 
+def relogin_if_needed(f):
+    """Repeat RT call after explicit login, if needed."""
```

Comment:
There's no involved login error. It's just a generic call to RT that fails due to expiration of something.